apps/bestofjs-nextjs/src/app/page.tsx

```diff
@@ -265,10 +265,8 @@ function MoreProjectsSection({
           .
         </p>
         <p>
-          Data is updated from GitHub everyday, the last update was{" "}
-          <b>{fromNow(lastUpdateDate)}</b> (at{" "}
-          {timeOnlyFormat.format(lastUpdateDate)}
-          ).
+          Data is updated from GitHub every 24 hours, the last update was
+          <b>{fromNow(lastUpdateDate)}</b>.
```

Comment:
PR looks great! Just needs a space here between `the last update` and the `fromNow` call.